.github/workflows/test.yml

```
@@ -126,3 +126,35 @@ jobs:
         env:
           TAG: test
           MODULE: Tokens
+  cns-transport-test:
+    name: Run CNS-Transport Tests
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          path: backbone  
+      - uses: actions/checkout@v3
+        with:
+          repository: nmshd/cns-transport
+          path: cns-transport
+      - name: Docker Compose build
```

I would name them in a more describing way, instead of repeating the command
```suggestion
      - name: Build Consumer API
```